src/components/ButtonGroup.js

```diff
@@ -0,0 +1,101 @@
+const React = require('react');
+const Radium = require('radium');
+
+const Button = require('./Button');
+
+const StyleConstants = require('../constants/Style');
+
+const ButtonGroup = React.createClass({
+  propTypes: {
+    buttons: React.PropTypes.arrayOf(React.PropTypes.shape({
+      icon: React.PropTypes.string,
+      text: React.PropTypes.string
+    }).isRequired),
+    controlArrows: React.PropTypes.bool,
+    primaryColor: React.PropTypes.string,
+    type: React.PropTypes.oneOf([
+      'base',
+      'disabled',
+      'neutral',
+      'primary',
+      'primaryOutline'
+    ])
+  },
+
+  getDefaultProps () {
+    return {
+      buttons: [],
+      controlArrows: false,
+      primaryColor: StyleConstants.Colors.PRIMARY,
+      type: 'primaryOutline'
+    };
+  },
+
+  _getArrows () {
+    this.props.buttons.unshift({ icon: 'caret-left' });
+    this.props.buttons.push({ icon: 'caret-right' });
+  },
+
+  render () {
+    const styles = this.styles();
+    const buttonType = this.props.type;
+    const primaryColor = this.props.primaryColor;
```

Comment:
I probably wouldn't pull `buttonType` or `primaryColor` in a const because you're only using them once here. There's no duplication.